tests/test_visitors/test_ast/test_complexity/test_counts/test_yield_length.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.visitors.ast.complexity.counts import (
+    TooLongYieldTupleViolation,
+    YieldTupleVisitor,
+)
+
+single_yield = """
+def function_name():
+    i = 0
+    while True:
+        yield i
+        i = i + 1
+"""
+
+short_yield = """
+def function_name():
+    i = 0
+    while True:
+        yield i + 1, i + 2, i + 3
+"""
+
+long_yield = """
+def function_name(foo, bar, baz):
+    i = 0
+    while True:
+        yield i + 1, i + 2, i + 3, foo + 1, bar + 1, baz + 1
```

Comment:
Let's do a template instead! And insert values into it.
Because `short_yield` and `long_yield` are already equal.

We should also test: `()` (empty tuple), `(1, 2, 3, 4, 5, 6)` (tuple with `(` and `)`)